### PR TITLE
Remove status from awaiting_determination

### PR DESCRIPTION
**Pull request by melzreal**
### Description of change
https://trello.com/c/0Id6IVnc/686-remove-status-column-on-awaiting-determination-tab

### CODE REVIEW

File: app/views/planning_applications/_planning_application_table.html.erb [hunk 1]
```diff
@@ -3,18 +3,8 @@
...
-      <th scope="col" class="govuk-table__header">Expiry date</th>
-      <% if id != "closed" %>
-        <th scope="col" class="govuk-table__header">Days left to expiry</th>
-      <% end %>
-      <th scope="col" class="govuk-table__header">Status</th>
-      <% if id == "awaiting_determination" %>
-        <th scope="col" class="govuk-table__header">Recommendation date</th>
-      <% elsif id == "closed" %>
-        <th scope="col" class="govuk-table__header">Determination date</th>
+      <% header_options(id).each do |option| %>
```
Comment (benbaumann95) [inline app/views/planning_applications/_planning_application_table.html.erb:6]: I know this existed already in the code but it's a bit confusing to have "id" mean the "status" here imo. Should we update the rendering of these partials to reflect what this really is.

File: app/helpers/index_helper.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+module IndexHelper
+  def header_options(id)
+    options = [
+      "Application number",
+      "Site address",
+      "Application type",
+      "Expiry date",
+      "Days left to expiry",
+      "Status"
+    ]
```
Comment (benbaumann95) [inline app/helpers/index_helper.rb:12]: Could be a constant?

**Review by benbaumann95 — COMMENTED** [review]
Overall like what you're trying to do here but just a few comments

File: app/helpers/index_helper.rb (hunk 2 above)
Comment (melzreal) [reply app/helpers/index_helper.rb:12]: I'm modifying it though in the next few lines though, isn't it a bit weird to be modifying a constant?

File: app/views/planning_applications/_planning_application_table.html.erb (hunk 1 above)
Comment (melzreal) [reply app/views/planning_applications/_planning_application_table.html.erb]: you're right, id is confusing - I've changed it to "planning_application_status" so its more readable

**Review by benbaumann95 — APPROVED** [review]
I think the helper could be more appropriately named, now that we've updated to reflect these are statuses and not id's.
Otherwise looks good!

**Pull request merged**